payroll-processor-client/src/app/employee/employee-list/state/employee-list.service.ts

```diff
@@ -0,0 +1,36 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+
+import { Employee } from './employee-list.model';
+import { EmployeeListStore } from './employee-list.store';
+
+import { EnvService } from 'src/app/shared/env.service';
+
+@Injectable({ providedIn: 'root' })
+export class EmployeeListService {
+  private readonly apiUrl: string;
+  private employees: Employee[];
+
+  constructor(
+    private http: HttpClient,
+    envService: EnvService,
+    private employeeListStore: EmployeeListStore,
```

Comment:
A lot of times I'll keep the dependency names simple until I need to be more specific.

So, here I'd name this `store` instead of `employeeListStore`.

Reply:
I wondered about this, I went specific because atleast I was being descriptive, but it felt heavy since in a lot of cases there is only 1 service or 1 store being injected right now

Reply:
In C# I still tend to be specific (probably just habit), but I've started moving away from that in Angular code. It's noisy without much benefit.

I'm more ok with changing the names of class/variables/files as the app grows as opposed to following a specific set of rules from the start.

Reply:
Let's keep them simple in the angular app and grow more verbose as the app increases in size. I'm okay with different naming conventions in the functions vs client, just keep an eye out on PRs for consistencies sake.